dev-packages/ext-scripts/package.json

```diff
@@ -4,27 +4,20 @@
   "version": "0.14.0",
   "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
   "description": "NPM scripts for Theia packages.",
-  "files": [
-    "theiaext"
-  ],
   "bin": {
-    "theiaext": "theiaext",
-    "run": "theia-run"
+    "theiaext": "theia-ext.js",
+    "run": "theia-run.js"
   },
-  "scripts": {
-    "prepare": "echo 'skip'",
-    "clean": "echo 'skip'",
-    "build": "echo 'skip'",
-    "watch": "echo 'skip'",
-    "test": "echo 'skip'",
-    "ext:clean": "theiaext compile:clean && theiaext test:clean",
+  "theia-monorepo-scripts": {
+    "ext:clean": "theiaext compile:clean && theiaext lint:clean && theiaext test:clean",
     "ext:build": "concurrently -n compile,lint -c blue,green \"theiaext compile\" \"theiaext lint\"",
-    "ext:compile": "tsc -p compile.tsconfig.json",
-    "ext:compile:clean": "rimraf lib",
-    "ext:lint": "tslint -c ../../configs/build.tslint.json --project compile.tsconfig.json",
-    "ext:watch": "tsc -w -p compile.tsconfig.json",
+    "ext:compile": "tsc -b compile.tsconfig.json --verbose",
```

Comment:
`verbose` is necessary?

Reply:
Depends, I wish the output was better than what it is currently:

- verbose: tells you what is being skipped and what is being built.
- no verbose: no output at all, you just wait and hope for the best.

Tell me what you think is best, as I don't really mind.

Reply:
ok, let's stick with verbose